win_etw_macros/src/lib.rs

```diff
@@ -616,23 +655,60 @@ fn trace_logging_events_core(attr: TokenStream, item_tokens: TokenStream) -> Tok
         impl #provider_ident {
             #provider_impl_items
         }
-
-        #[doc(hidden)]
-        #[allow(non_snake_case)]
-        mod #provider_mod_ident {
-            /*
-            pub(crate) mod event_descriptors {
-                #![allow(non_upper_case_globals)]
-                #( #event_descriptors )*
-            }
-            */
-        }
     });
 
     output.extend(errors.into_iter().map(|e| e.to_compile_error()));
     output
 }
 
+/// Creates a fragment of code (statements) which will register the
+/// provider traits for this provider.
+fn create_register_provider_traits(
```

Comment:
Could we add a link to any documentation about the format used below?

Reply:
There's already a comment at the call site to `create_register_provider_traits`. I'll move that to the func def.